format.sh

```diff
@@ -77,7 +81,7 @@ format_changed() {
     MERGEBASE="$(git merge-base origin/master HEAD)"
 
     if ! git diff --diff-filter=ACM --quiet --exit-code "$MERGEBASE" -- '*.py' '*.pyi' &>/dev/null; then
-        git diff --name-only --diff-filter=ACM "$MERGEBASE" -- '*.py' '*.pyi' | xargs -P 5 \
+        git diff --name-only --diff-filter=ACM "$MERGEBASE" -- '*.py' '*.pyi' | xargs -P 5 -d '\n' \
```

Comment:
Looks like `-d` is not available on mac:
```
xargs: invalid option -- d
usage: xargs [-0opt] [-E eofstr] [-I replstr [-R replacements] [-S replsize]]
             [-J replstr] [-L number] [-n number [-x]] [-P maxprocs]
             [-s size] [utility [argument ...]]
```

Any other fix than having all macos users install gnu xargs? https://superuser.com/questions/467176/replacement-for-xargs-d-in-osx

Reply:
Thanks for the catch! This only matters if there are filenames with spaces, which we probably don't expect but _should_ still handle.
I think we can use `tr '\n' '\0' | xargs -0` instead. Will update.